src/Analyzers/Analyzers.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsRoslynComponent>true</IsRoslynComponent>
```

Comment:
Oh, can you add `<IsPackable>false</IsPackable>` until this is ready to release? Don't want this to accidentally be included in a release for now.